Components/Sources/Components/Components/Watchlist/WKWatchlistViewController.swift

```diff
@@ -5,10 +5,20 @@ import WKData
 
 public protocol WKWatchlistDelegate: AnyObject {
 	func watchlistDidDismiss()
-    func emptyViewDidTapSearch()
 	func watchlistUserDidTapDiff(project: WKProject, title: String, revisionID: UInt, oldRevisionID: UInt)
 	func watchlistUserDidTapUser(project: WKProject, username: String, action: WKWatchlistUserButtonAction)
+    func emptyViewDidTapSearch()
```

Comment:
Fixed here, and I also removed `watchlistDidDismiss` since it wasn't used. Feel free to push any additional naming tweaks you'd like to this PR (or in the next PR) if you want!